packages/aws-client/S3.js

```diff
@@ -271,9 +271,26 @@ exports.getS3Object = improveStackTrace(
     )
 );
 
-exports.getJsonS3Object = (bucket, key) =>
+/**
+ * Fetch the contents of an S3 object
+ *
+ * @param {string} bucket - the S3 object's bucket
+ * @param {string} key - the S3 object's key
+ * @returns {Promise<string>} the contents of the S3 object
+ */
+exports.getTextObject = (bucket, key) =>
   exports.getS3Object(bucket, key)
-    .then(({ Body }) => JSON.parse(Body.toString()));
+    .then(({ Body }) => Body.toString());
+
+/**
+ * Fetch JSON stored in an S3 object
+ * @param {string} bucket - the S3 object's bucket
+ * @param {string} key - the S3 object's key
+ * @returns {Promise<*>} the contents of the S3 object, parsed as JSON
```

Comment:
The type of the return object should be a string, right?

Reply:
It's being sent through `JSON.parse`, so it could be an object, array, string, number, boolean, etc.

Reply:
TIL: JSON.parse can return other values. OK.